### PR TITLE
Change "product" to "Product"

### PR DESCRIPTION
**Pull request by apinnick**
Changed "product" to "Product" to align with RHSSG. Not sure whether we do the same for SUSE.

* [x] I am okay with my commits getting squashed when you merge this PR.
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.11/Katello 4.13
* [ ] Foreman 3.10/Katello 4.12
* [ ] Foreman 3.9/Katello 4.11 (Satellite 6.15; orcharhino 6.8/6.9)
* [ ] Foreman 3.8/Katello 4.10
* [ ] Foreman 3.7/Katello 4.9 (Satellite 6.14)
* [ ] Foreman 3.6/Katello 4.8
* [ ] Foreman 3.5/Katello 4.7 (Satellite 6.13; orcharhino 6.6/6.7)
* We do not accept PRs for Foreman older than 3.5.

### CODE REVIEW

**Review by ekohl — COMMENTED** [review]
As I mentioned in private: I think we should treat `product` the same as `content view` and `lifecycle environment`. Perhaps this is my engineer's view, but all of them are models in Katello. Since it was decided that `content view` and `lifecycle environment` should be lower cased, I think `product` should also be lower case.

File: guides/common/modules/proc_adding-an-scc-account.adoc [hunk 1]
```diff
@@ -20,23 +20,23 @@ Add the GPG public key for SLES 15 SP3 from https://www.suse.com/support/securit
 By default, it is set to `\https://scc.suse.com`.
 . Optional: Set a *Sync interval* to periodically update the SCC authentication tokens.
 Note that this does not refer to synchronizing content to {Project}.
-. Optional: In the *Use GPG key for SUSE products* field, you can select the previously created content credential to automatically add a GPG public key to your SUSE products.
+. Optional: In the *Use GPG key for SUSE products* field, you can select the previously created content credential to automatically add a GPG public key to your SUSE Products.
```
Comment (asteflova) [inline guides/common/modules/proc_adding-an-scc-account.adoc:23]: Stating the obvious: product/Product is capitalized inconsistently on this line. I don't have a VM with the right plug-in available so I can't check the context but it does look like "Product" is the right choice here too. Perhaps we could change the label of that field in the web UI? I suppose the PR should be simple, except that I can't find the right place in the code.

**Review by maximiliankolb — APPROVED** [review]
diff LGTM

**Comment by maximiliankolb** [comment]
> As I mentioned in private: I think we should treat `product` the same as `content view` and `lifecycle environment`. Perhaps this is my engineer's view, but all of them are models in Katello. Since it was decided that `content view` and `lifecycle environment` should be lower cased, I think `product` should also be lower case.

While reviewing this PR, I had the same feeling (the other way around: capitalize repository, CV, and LE etc). Some sentences that contain "Product" and any other entity such as repository feel weird to read and make you wonder if there's a mistake in capitalization.

However, I am still OK with this change.

**Comment by asteflova** [comment]
Looking into the [PatternFly style guide](https://www.patternfly.org/ux-writing/capitalization#red-hat-product-uis):

> Capitalize product feature names when they’re used as proper nouns or when they refer to a capitalized UI term (like a navigation item). Write them in lowercase when they’re used to describe generic concepts. For example:

> “Add a source by going to Settings > Sources.” "Check the Sources table for status."
> 
> Button text: "Add source"
> 	
> “Sources” is only capitalized when it directly refers to a subsection, feature, or location in the UI. "Source" is lowercase in the button text because button labels should always be in sentence case.

If I'm reading this right, it would mean lowercase for product/LE/CV/repository when used in regular sentences in docs.

**Comment by apinnick** [comment]
I am closing this PR pending discussion about Product vs product.

**Pull request closed without merging**